app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -321,15 +321,17 @@ public void setRightCardVisibility(boolean visible) {
     @Override
     public void setBottomCardVisibility(@UploadPage int page, int uploadCount) {
         if (page == TITLE_CARD) {
+            setContentView(R.layout.activity_upload);
```

Comment:
why is this required?

Reply:
At the very first time when value of page will be 0(i.e PLEASE_WAIT) thus it will set the contentview to R.layout.activity_upload_please_wait the next time page comes with any value we need to first set the content view to R.layout.activity_upload and then use the view flipper. This was the logic over which I had worked upon. But curently I think there can be a better solution than this.

Can you please give some more time to come up with a better solution

Reply:
`activity_upload_please_wait` is also part of the `ViewFlipper`. You basically need to hide the other elements ie `activity_upload_top_card` and `activity_upload_right_card` so that the ViewFlipper can take up the whole space. 

Infact, if you go through the upload flow, you would notice that  `activity_upload_top_card` and `activity_upload_right_card` are visible only along with `activity_upload_bottom_card`. It would be logical to combine them as a single child of the `ViewFlipper`. That way the `ViewFlipper` can always `match_parent`.

Reply:
I have made the following changes:
Changed attribute of ViewFlipper to match-parent
copied the content of `activity_upload_top_card` and `activity_upload_right_card` to `activity_upload_bottom_card`
Kindly review and share feedback's :)

Reply:
It looks like you forgot to push your changes.

Reply:
I am sorry. Actually I have changed the branch, that was the problem. I have send another PR here https://github.com/commons-app/apps-android-commons/pull/2370